### ISSUE TITLE
Can't create playlist

### ISSUE THREAD

**Issue by UnnamedHero** [libasys/audios#12]
Can't create playlist.
OS: Centos 7.1.1506
OC: 8.1.1
DB: PostgreSQL
App version 1.2.4
 After pressing [GO] button I've got this in OC logs:

```
Doctrine\DBAL\Exception\DriverException: An exception occurred while executing 'SELECT "id", COUNT("id") AS COUNTID FROM "oc_audios_playlists" WHERE "user_id" = ? AND "name" = ?' with params ["USERNAME", "Therion"]: SQLSTATE[42803]: Grouping error: 7 ERROR: column "oc_audios_playlists.id" must appear in the GROUP BY clause or be used in an aggregate function LINE 1: SELECT "id", COUNT("id") AS COUNTID FROM "oc_audios_playlis... ^
/mnt/sdc/www/qqo/3rdparty/doctrine/dbal/lib/Doctrine/DBAL/DBALException.php - line 116: Doctrine\DBAL\Driver\AbstractPostgreSQLDriver->convertException('An exception oc...', Object(Doctrine\DBAL\Driver\PDOException))
/mnt/sdc/www/qqo/3rdparty/doctrine/dbal/lib/Doctrine/DBAL/Statement.php - line 172: Doctrine\DBAL\DBALException driverExceptionDuringQuery(Object(Doctrine\DBAL\Driver\PDOPgSql\Driver), Object(Doctrine\DBAL\Driver\PDOException), 'SELECT "id", CO...', Array)
/mnt/sdc/www/qqo/lib/private/db/statementwrapper.php - line 73: Doctrine\DBAL\Statement->execute(Array)
/mnt/sdc/www/qqo/apps/audios/controller/playlistcontroller.php - line 108: OC_DB_StatementWrapper->execute(Array)
/mnt/sdc/www/qqo/apps/audios/controller/playlistcontroller.php - line 86: OCA\Audios\Controller\PlaylistController->writePlaylistToDB('Therion')
[internal function] OCA\Audios\Controller\PlaylistController->addPlaylist()
/mnt/sdc/www/qqo/lib/private/appframework/http/dispatcher.php - line 159: call_user_func_array(Array, Array)
/mnt/sdc/www/qqo/lib/private/appframework/http/dispatcher.php - line 89: OC\AppFramework\Http\Dispatcher->executeController(Object(OCA\Audios\Controller\PlaylistController), 'addPlaylist')
/mnt/sdc/www/qqo/lib/private/appframework/app.php - line 108: OC\AppFramework\Http\Dispatcher->dispatch(Object(OCA\Audios\Controller\PlaylistController), 'addPlaylist')
/mnt/sdc/www/qqo/lib/private/appframework/routing/routeactionhandler.php - line 45: OC\AppFramework\App main('PlaylistControl...', 'addPlaylist', Object(OC\AppFramework\DependencyInjection\DIContainer), Array)
[internal function] OC\AppFramework\routing\RouteActionHandler->__invoke(Array)
/mnt/sdc/www/qqo/lib/private/route/router.php - line 274: call_user_func(Object(OC\AppFramework\routing\RouteActionHandler), Array)
/mnt/sdc/www/qqo/lib/base.php - line 873: OC\Route\Router->match('/apps/audios/ad...')
/mnt/sdc/www/qqo/index.php - line 40: OC handleRequest()
{main}

```

**Comment by UnnamedHero** [comment]
Not fixed in 1.2.5

**Comment by UnnamedHero** [comment]
Fixed in master branch
